BuildRelease.msbuild

```diff
@@ -104,7 +104,7 @@
         <ItemGroup>
           <Xunit1TestAssemblies
               Include="Src\*Test\bin\$(Configuration)\*Test.dll"
-              Exclude="Src\AutoFixture.NUnit2.*Test\bin\$(Configuration)\*.dll" />
+              Exclude="Src\AutoFixture.xUnit.net2.UnitTest\bin\$(Configuration)\Ploeh.AutoFixture.Xunit2.UnitTest.dll;Src\AutoFixture.NUnit2.*Test\bin\$(Configuration)\*.dll" />

```

Comment:
Can you elaborate on why this exclusion is necessary and how you intend to move forward?

While I did see the interchange regarding the build issue, it only informs me of the superficial reason for excluding the new unit test library.

However, I thought that the reason to add the new xUnit.net 2 runners (in a previous pull request) was to add support for also running xUnit.net 2 unit test libraries. Will this be possible with the build-release script, and if so, why not do whatever it takes first?